crime-applications-adaptor/api-tests/build.gradle

```diff
@@ -9,7 +9,7 @@ def versions = [
         jUnitPlatformSuiteVersion: "1.10.2",
         cucumberEngineVersion    : "7.17.0",
         junitBOMVersion          : "5.10.2",
-        commonsVersion           : "0.12.0"
+        commonsVersion: "1.2.1"
```

Comment:
Minor, can you tidy up the whitespace formatting so the : aligns?

Reply:
Spotless Gradle plugin is reformatting this if we align the format.